Sources/BraintreeVenmo/BTVenmoRequest.swift

```diff
@@ -41,6 +41,38 @@ import Foundation
 
     /// Optional. The business name that will be displayed in the Venmo app payment approval screen. Only used by merchants onboarded as PayFast channel partners.
     public var displayName: String?
+    
+    /// Whether the customer's billing address should be collected and displayed on the Venmo paysheel.
+    /// Defaults to `false`
+    public var collectCustomerBillingAddress: Bool = false
+    
+    /// Whether the customer's shipping address should be collected and displayed on the Venmo paysheel.
+    /// Defaults to `false`
+    public var collectCustomerShippingAddress: Bool = false
+    
+    /// Optional.
+    /// The subtotal amount of the transaction to be displayed on the paysheet. Excludes taxes, discounts, and shipping amounts.
```

Comment:
Nitpicks for consistency

```suggestion
    /// Optional. The subtotal amount of the transaction to be displayed on the paysheet. Excludes taxes, discounts, and shipping amounts.
```